api/src/main/java/org/eclipse/microprofile/rest/client/spi/RestClientBuilderResolver.java

```diff
@@ -117,12 +85,8 @@ public static RestClientBuilderResolver instance() {
                     return instance;
                 }
 
-                ClassLoader cl = AccessController.doPrivileged(new PrivilegedAction<ClassLoader>() {
-                    @Override
-                    public ClassLoader run() {
-                        return Thread.currentThread().getContextClassLoader();
-                    }
-                });
+                ClassLoader cl = AccessController.doPrivileged(
+                    (PrivilegedAction<ClassLoader>) () -> Thread.currentThread().getContextClassLoader());
```

Comment:
My IDE also recommended this but I'm not sure it's more readable :) 

It's probably nicer to assign the lambda to a local variable, like this:
```
PrivilegedAction<ClassLoader> action = () -> Thread.currentThread().getContextClassLoader();
ClassLoader cl = AccessController.doPrivileged(action);
```

Reply:
Agreed